common/gckms/GckmsConfig.js

```diff
@@ -10,8 +10,6 @@ const publicNetworkNames = Object.values(require("../PublicNetworks.js")).map(el
 
 let configOverride = {};
 
-console.log(process.env.GCKMS_CONFIG);
```

Comment:
@mrice32 I think you left this console.log here unintentionally?

Reply:
ye, looks like that should be removed.

Reply:
Yep, my bad.